src/Components/AppLifecycleTable/appLifecycleTable.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Table, Tbody, Tr, Td, Thead, Th } from '@patternfly/react-table';
+import Moment from 'moment';
+
+interface dataAppLifeCycleChanges {
+    data: AppLifecycleChanges[];
+
+}
+
+interface AppLifecycleChanges {
+    module_name: string;
+    rhel_major_version: number;
+    streams: Stream[];
+  }
+  
+interface Stream {
+    arch: string;
+    context: string;
+    description: string;
+    end_date: string;
+    name: string;
+    profiles: Profiles;
+    start_date: string;
+    stream: string;
+    version: string;
+  }
+  
+interface Profiles {
+    common: string[];
+  }
+
+const columnNames = {
+  name: 'Name',
+  release: 'Release',
+  rhel_version: 'RHEL Version',
```

Comment:
```suggestion
  rhel_version: 'Release',
```
Following what's in the Figma mocks

Reply:
Fixed, thanks! I completely reoved the `rhel_version` and used the release